src/includes/performance/retrieve-transaction/rust.mdx

```diff
@@ -0,0 +1,21 @@
+## Retrieve a Transaction / Span
+
+In cases where you want to attach Spans to an already ongoing Transaction you can use `Scope::get_span`. This method will return a `TransactionOrSpan` in case there is a running Transaction or Span, otherwise it returns `None`.
```

Comment:
```suggestion
In cases where you want to attach spans to an already ongoing transaction, you can use `Scope::get_span`. This method will return a `TransactionOrSpan` in case there is a running transaction or span; otherwise it returns `None`.
```